components/automate-cli/pkg/verifyserver/utils/checkutils/checkutil_test.go

```diff
@@ -68,9 +69,9 @@ func TestPrepareTriggerResponse_WithError(t *testing.T) {
 	check := "test check"
 	msg := "test message"
 	isError := true
-
+	statusCode := 400
 	// Act
-	result := PrepareTriggerResponse(resp, host, nodeType, errorString, check, msg, isError)
+	result := PrepareTriggerResponse(resp, host, nodeType, errorString, check, msg, isError, statusCode)
```

Comment:
```suggestion
	result := PrepareTriggerResponse(resp, host, nodeType, errorString, check, msg, isError, fiber.StatusBadRequest)
```
you can directly use like this